src/Sidebar.jsx

```diff
@@ -7,17 +7,17 @@ import Profile from "./Images/profile logo.png";
 import { SidebarItems } from "./ConstantData";
 
 export default function Sidebar() {
-  const [Sidebar, setSidebar] = useState(true);
+  const [SidebarWidth, setSidebarWidth] = useState(true);
   const [CurrentSelectedItem, setCurrentSelectedItem] = useState(null);
 
   const ToggleSidebar = () => {
-    setSidebar(!Sidebar);
+    setSidebarWidth(!SidebarWidth);
   };
   return (
     <div>
       <nav
         className={`bg-gray-300 fixed h-full rounded-lg ${
-          Sidebar ? "w-36 sm:w-20 md:w-36 duration-500" : "w-20 duration-500"
+          SidebarWidth ? "sm:w-24 md:w-28 lg:w-40 xl:w-48 2xl:w-56 duration-500" : "w-20 md:w-24 lg:w-32 duration-500"
```

Comment:
If you want to apply same styling for all screen sizes, you just use the style without mentioning the screen size. So, no need for sm before style.

Reply:
its changing on the condition , Styling is not same as width will be changed on different screen sizes and width of the sidebar will change